vnext/ReactUWP/Views/ViewManagerBase.cpp

```diff
@@ -13,6 +13,7 @@
 #include <IXamlRootView.h>
 #include <Views/ShadowNodeBase.h>
 #include <winrt/Windows.UI.Xaml.h>
+#include "Views/ReactControl.h"
```

Comment:
Is this needed? Only change in the file.

Reply:
no.

---
In reply to: [299687406](https://github.com/microsoft/react-native-windows/pull/2706#discussion_r299687406) [](ancestors = 299687406)